### PR TITLE
Bug 1117115 - Enable test_settings_sim_manager.py

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[JohanLorenzo Johan Lorenzo (JohanLorenzo)](https://github.com/JohanLorenzo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e158c0fe3734)

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini [hunk 1]
```diff
@@ -80,8 +80,7 @@ online = true
 skip-if = device == "desktop" && os == "linux" && debug
 
 [test_settings_sim_manager.py]
-disabled = Disabled because run the device with 1 SIM only
-run-if = device == "sp7710gaplus_gonk" || device == "flame"
+dsds = true
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini]: Doesn't this also need carrier = true?

Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini]: Currently, DSDS implies that you have 2 SIM cards and we only need that in this test. Carrier is not necessary here then.

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/sim_manager.py [hunk 2]
```diff
@@ -15,54 +15,38 @@ class SimManager(Base):
...
+    def select_outgoing_messages(self, sim_option):
         self.marionette.find_element(*self._outgoing_messages_locator).tap()
-        self.select('SIM '+str(sim))
+        self.select(sim_option)
 
-    def select_data(self, sim):
+    def select_data(self, sim_option):
         self.marionette.find_element(*self._outgoing_data_locator).tap()
-        self.select_and_confirm('SIM '+str(sim))
+        self.select(sim_option)
```
Comment (viorelaioia) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/sim_manager.py]: :+1:

**Comment by viorelaioia** [comment]
Lgtm, I ran the test 10 times on a device with 2 SIMs and it worked as expected.

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini (hunk 1 above)
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini]: To the test runner no attributes imply other attributes. They must be explicit.

**Pull request merged**